### ISSUE TITLE
The background image of the website homepage supports CDN addresses

### ISSUE THREAD

**Issue by misitebao** [hugo-toha/toha#572]
### Describe the solution you'd like

It is hoped that the background image of the homepage of the website can support the CDN address, so that the acceleration function of CDN such as `jsdelivr` can be fully utilized to solve the problem that the website server accesses pictures slowly.

**Comment by hossainemruz** [comment]
Did you tried providing a URL in the `params.background` field? It should work already.

**Comment by misitebao** [comment]
No, I couldn't use cdn when I tested it last year. I just re-tested it, but it still couldn't, and the same error was prompted.
![image](https://user-images.githubusercontent.com/28185258/160606850-8623c160-4180-45ee-bc3a-b39b770db7e2.png)

**Comment by hossainemruz** [comment]
What is your Hugo version?

**Comment by misitebao** [comment]
I tested the 0.88.0 and 0.96.0 versions separately, and the results were the same.

**Comment by hossainemruz** [comment]
Ok. Then,we have to add the support. I will take a look soon.
